tests/test_process.py

```diff
@@ -221,10 +221,48 @@ def test_process_list_widget(multiply_add_completed_workchain):
     ProcessListWidget()
 
 
+@pytest.mark.usefixtures("aiida_profile_clean")
+def test_process_monitor(
+    multiply_add_process_builder_ready, daemon_client, await_for_process_completeness
+):
+    """Test ProcessMonitor with a simple `WorkChainNode`"""
+    from aiidalab_widgets_base.process import ProcessMonitor
+
+    ProcessMonitor()
+
+    # Stopping the daemon and submitting the process.
+    daemon_client.stop_daemon(wait=True)
+    process = engine.submit(multiply_add_process_builder_ready)
+
+    test_variable = False
+
+    def f():
+        nonlocal test_variable
+        test_variable = True
+
+    widget = ProcessMonitor(value=process.uuid, callbacks=[f])
+
+    # Starting the daemon and waiting for the process to complete.
+    daemon_client.start_daemon()
+    await_for_process_completeness(process)
+
+    widget.join()  # Make sure the thread is finished.
+
+    assert test_variable
+
+    # Now do the same with the process trait instead of value.
+    daemon_client.stop_daemon(wait=True)
+    process = engine.submit(multiply_add_process_builder_ready)
+    ProcessMonitor(process=process)
+    daemon_client.start_daemon()
+    await_for_process_completeness(process)
+
```

Comment:
```suggestion
    daemon_client.stop_daemon(wait=True)
```

If use `daemon_client`, need to stop daemon explicitly after the test.